docs/tutorial/connect/create-connected-rows.md

```diff
@@ -119,7 +119,7 @@ Let's start by creating two teams:
 
 ///
 
-This would hopefully look already familiar.
+This would hopefully looks already familiar.
```

Comment:
This change would apply if the phrase was:

> This hopefully looks

But having the `would` there then it should be:

> This would hopefully look

...can you please revert this change?

Reply:
Opsie, my bad :smile: done